modules/home/assets/js/components/top-section.js

```diff
@@ -0,0 +1,44 @@
+import { Box, Paper } from '@elementor/ui';
+import Typography from '@elementor/ui/Typography';
+import Button from '@elementor/ui/Button';
+import YoutubeIcon from './icons/youtube-icon';
+const TopSection = ( { ctaUrl, videoUrl, embedUrl } ) => {
+	return (
+		<Paper justifyContent="space-between" alignItems="flex-start" elevation={ 0 } sx={ { display: 'flex', flexDirection: 'row', bgcolor: 'theme.palette.background.default', maxWidth: 'lg', mx: 'auto', mt: 6 } }>
+			<Box ml={ 4 } my={ 4 }>
+				<Box sx={ { display: 'flex', flexDirection: 'column', width: '85%', mb: 3 } }>
+					<Typography variant="body2">Hi!</Typography>
+					<Typography variant="h6">Welcome to Elementor</Typography>
+					<Typography variant="body2" >Get introduced to Elementor by watching our &ldquo;Getting Started&rdquo; video series. It will guide you through the steps needed to create your website. Then click to create your first page.</Typography>
+				</Box>
+				<Box>
+					<Button variant="contained" size="small" sx={ { mr: 1 } } href={ ctaUrl } target="_blank">
+						<Typography variant="button">Create a Page</Typography>
+					</Button>
+					<Button variant="outlined" color="secondary" size="small" startIcon={ <YoutubeIcon /> } href={ videoUrl } target="_blank">
```

Comment:
@ronenelementor Can you please check if we have this icon in the Icon Library already?

Where can we find the Elementor V2 icon library again? I know @rotemee showed it to us, but I can't remember where I can find the icons.

Reply:
I don't see any icon for youtube in elementor-ui or mui, there is a class name that works 'eicon-youtube' though

Reply:
@ronenelementor I think that we should use the old Icon Library and its class names. Rotem and his team are building an icon library for the V2 design library.

Reply:
talked to Hadar, for MUI we need to use svgs.